.rubocop.yml

```diff
@@ -42,6 +42,12 @@ Naming/PredicateName:
 Style/CaseEquality:
   Enabled: false
 
+Style/GlobalVars:
+  AllowedVariables:
+    - $CFLAGS
+    - $INCFLAGS
+    - $libs
```

Comment:
I think I'd rather just

```yml
Style/GlobalVars:
  Exclude:
    - 'ext/geos_c_impl/extconf.rb'
```

Since we don't want those variables somewhere else!